src/ui/buttons/buttons.jsx

```diff
@@ -278,8 +272,8 @@ export function Buttons(props: ButtonsProps): ElementNode {
         fundingEligibility={fundingEligibility}
       />
 
-      {message && calculatedMessagePosition === MESSAGE_POSITION.TOP ? (
-        <Message markup={messageMarkup} position={calculatedMessagePosition} />
+      {message && message.position === MESSAGE_POSITION.TOP ? (
+        <Message markup={messageMarkup} position={message.position} />
```

Comment:
good thought - the effect is that flow complains about the `position = {message.position}`:

> Cannot get `message.position` because property `position` is missing in  undefined [1].Flow(incompatible-use)

Reply:
Oh right. Thanks for the reminder hah